kubernetes/doc/edge-deployment-to-k8s-translations.md

```diff
@@ -0,0 +1,215 @@
+# Conversion of Edge Deployment ModuleSpec to Kubernetes Deployments.
+
+Azure IoT Edge on Kubernetes project will take an existing Azure IoT Edge application that is 
```

Comment:
Mention of "projects" and "applications" isn't required because we deploy Edge devices (not applications) right?
```suggestion
Azure IoT Edge supports deploying an IoT Edge device on a Kubernetes cluster, as well as a single Docker instance. 
```

Reply:
Pulled in the other changes, but I reworded this more extensively.